src/org/opensolaris/opengrok/history/GitRepository.java

```diff
@@ -675,15 +676,27 @@ String determineCurrentVersion() throws IOException {
         cmd.add("-1");
         cmd.add("--pretty=%cd: %h %an %s");
         cmd.add("--date=format:%Y-%m-%d %H:%M");
+
         ProcessBuilder pb = new ProcessBuilder(cmd);
         pb.directory(directory);
-        Process process;
-        process = pb.start();
+        Process process = pb.start();
 
-        try (BufferedReader in = new BufferedReader(new InputStreamReader(process.getInputStream()))) {
-            if ((line = in.readLine()) != null) {
-                line = line.trim();
+        try {
+            process.waitFor(15, TimeUnit.SECONDS);
```

Comment:
I have merged, but I have some doubts that this will be a problem for long first inits of git repo, no?
will retest and worst case open a bug against this